dpnp/dpnp_array.py

```diff
@@ -483,40 +513,51 @@ def __repr__(self):
         return dpt.usm_ndarray_repr(self._array_obj, prefix="array")
 
     def __rfloordiv__(self, other):
+        """Return ``value//self``."""
         return dpnp.floor_divide(self, other)
 
     def __rlshift__(self, other):
+        """Return ``value<<self``."""
         return dpnp.left_shift(other, self)
 
     def __rmatmul__(self, other):
+        """Return ``value@self``."""
         return dpnp.matmul(other, self)
```

Comment:
Is it possible to have a test (and coverage) for it?

Reply:
No, I don't think so. Currently we are quite strict to supported dtypes in `dpnp.matmul` call. They can be either dpt.tensor or dpnp array. While `__rmatmul__` requires no `__matmul__` method defined for type of the first input argument.

And that first input can't be a scalar, because even if we cast it to 0-dimension array (as we usually did for other elementwise functions), it will not work with `matmul` since requires at least 1-d array.

While for numpy it will work in quite simple way:
```python
a = numpy.array([2])
[1] @ a
# Out: 2
```